theme/static/css/hydroshare_core.css

```diff
@@ -4256,6 +4273,22 @@ div.fb-upload-caption a {
 	font-size: 18px;
 }
 
+.fb-view-list .fb-stack {
+	width: 18px;
+	height: 18px;
```

Comment:
@mauriel133 I see ```.fb-stack``` class is defined with ```height: 32px``` above, and here it is redefined with 18px height. Is the ```height: 32px``` definition above really needed?